docs/website-v2/versioned_sidebars/version-1.4.0-sidebars.json

```diff
@@ -266,7 +266,7 @@
         },
         {
           "type": "doc",
-          "id": "version-1.4.0/howtos/header_enrichment.md"
+          "id": "version-1.4.0/howtos/he_api"
```

Comment:
Docusaurus catch sidebar items that were not found. You need to use the id frontmatter (if it has been set)